conda/environments/rmm_dev_cuda11.5.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
 - flake8=3.8.3
 - black=19.10
 - isort=5.6.4
-- python>=3.7,<3.9
+- python>=3.8,<3.10
```

Comment:
@shwina The current 22.04 branch also has an environment for CUDA 11.6. Maybe pull upstream and fix that too? https://github.com/rapidsai/rmm/tree/branch-22.04/conda/environments

Reply:
Yes let's get that into this PR as well.

Reply:
Done